plasmapy/physics/tests/test_distribution.py

```diff
@@ -42,3 +47,84 @@ def test_Maxwellian_1D():
 
     with pytest.raises(ValueError):
         Maxwellian_1D(1*u.m/u.s, T=1*u.K, particle='XXX')
+
+def test_Maxwellian_speed_1D():
+    r"""test the 1D Maxwellian speed distribution function"""
+    T = 1.0 * u.eV
+    particle = 'H'
+    # get thermal velocity and thermal velocity squared
+    vTh = thermal_speed(T, particle=particle, method="most_probable")
+    # setting up integration from 0 to 10*vTh
+    xData1D = np.arange(0, 10.01, 0.01) * vTh
+    yData1D = Maxwellian_speed_1D(v=xData1D,T=T,particle=particle)
+    # integrating, this should be close to 1
+    integ = spint.trapz(y=yData1D, x=xData1D)
+    print("1D integral {0}".format(integ))
+    exceptStr = "Integral of distribution function should be 1."
+    assert np.isclose(integ.value, 1), exceptStr
```

Comment:
How about this, using the power of python 3.6 (though a `.format` would of course work as well, except for being less _awesome_):

    # cut the print
    exceptStr = f"Integral of distribution function should be 1 and not {integ}."
    assert # as previously except it returns the value now

Reply:
oops, I'll delete the `print` line, that was mainly for my own testing. I like the idea of including the value in the exception string though!